### DISCUSSION TITLE
Minimum Collection Threshold Setting

### DISCUSSION THREAD

**Discussion by jocklover** [CodyBerenson/PGMA-Modernized#160] [Ideas]
Plex built-in new Movie Agent offers ability to set minimum collection size. 
See: https://support.plex.tv/articles/advanced-settings-plex-movie-agent/
         (The setting is found under Manage Library > Edit > Advanced)

Would it be possible to incorporate similar functionality to PGMA? I have encountered a number of key-word type collections being pulled down as well as more obscure actors (including Unknown Male ####).
I would be fine if this was built in to Agent or Library like Plex Movie.

**Comment by JPH71** [comment]
Will look at this soon....

**Reply by jfreiman** [reply]
I too would love if PGMA could implement the Collection Minimum threshold! 
Currently, I have HUNDREDS of Collections created for just one Adult Library and many of them only have 1 title in them.
There are so many Collections automatically created that they are not even usable with a 10ft interface using Plex or a similar client.
I even have most of the PGMA Agents turned off (unchecked) to minimize the clutter that is created.

Does anyone know if Collections can be sorted or listed by # of titles in a collection?  It would be great if I could sort this way so I could easily delete all the collections with 0-3 titles without having to scroll and scroll and scroll through the long list of Collections to delete them. ;)

**Comment by JPH71** [comment]
I have yet to find a way to handle minimum collection sizes...
As it is with the new smart collections I suggest we drop this collection
gathering and leave collection gathering to be on film series as it
currently stands in AEBN and Gayhotmovies....

Thus collections will be based just on the filename using
space..dash..space and if a website has this information....

**Reply by jocklover** [reply]
I would just want to ensure that an actor's filmography could still be pulled up. But I think would still happen w/ the Cast Collection tagging (?).
